AcademyResidentInformationApi.Tests/V1/Gateways/AcademyGatewayTests.cs

```diff
@@ -70,57 +68,226 @@ public void GetClaimantInformationByClaimIdAndPersonRefReturnsAddressDetails()
                 AddressLine4 = address.AddressLine4,
                 PostCode = address.PostCode,
             };
-            response.AddressList.Should().BeEquivalentTo(new List<DomainAddress> { expectedDomainAddress });
+            response.ClaimantAddress.Should().BeEquivalentTo(expectedDomainAddress);
         }
 
         [Test]
         public void GetAllClaimantsIfThereAreNoClaimantsReturnsAnEmptyList()
         {
-            _classUnderTest.GetAllClaimants().Should().BeEmpty();
+            _classUnderTest.GetAllClaimants(0, 20).Should().BeEmpty();
         }
 
         [Test]
         public void GetAllClaimantsIfThereAreClaimantsWillReturnThem()
```

Comment:
This doesn't seem to be testing what the title suggests. The assertion is checking that the list is empty, which is the same as the test above.

Reply:
There's not been any major change to the test, but the method now requires a cursor and limit variable.

Reply:
It looks like the test has changed from adding records to the database then checking they have all been returned to just checking that nothing has been returned.

Reply:
@elcorbs I've merged those tests together, as there will not be instances of housing benefit claimants who do not have an address, so demonstrating the claimant was returned could be combined with the test that they have been returned with an address.